test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml

```diff
@@ -1,9 +1,13 @@
 test_duration: 3100
-prepare_write_cmd: "cassandra-stress write cl=ALL n=200200300  -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
+prepare_write_cmd:
+  - "cassandra-stress write cl=ALL n=200200300  -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
+  - "cassandra-stress write cl=ALL n=500 -schema 'keyspace=lowload1 replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=50 -col 'size=FIXED(1024) n=FIXED(4)' -pop seq=1..500 -log interval=15"
```

Comment:
```suggestion
  - "cassandra-stress write cl=ALL n=500 -schema 'keyspace=lowload1 replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1 -col 'size=FIXED(50) n=FIXED(1)' -pop seq=1..500 -log interval=15"
```

Reply:
The rate threads is what makes it "low load".
I also changed the size of the columns so it will be a tiny table.

Reply:
Done